crates/bevy_tasks/src/task_pool.rs

```diff
@@ -299,8 +299,11 @@ impl TaskPool {
                     break result;
                 };
 
-                self.executor.try_tick();
-                task_scope_executor.try_tick();
+                std::panic::catch_unwind(|| {
+                    executor.try_tick();
+                    task_scope_executor.try_tick();
+                })
+                .ok();
```

Comment:
These seem unrelated. Can you revert these changes?

Reply:
This is a temporary fix for #6603